docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,13 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+DAML Assistant
+~~~~~~~~~~~~~~
+
+- Added `--install-assistant` flag to `daml install` command, changing the default
```

Comment:
sorry, rst
```suggestion
- Added ``--install-assistant`` flag to ``daml install`` command, changing the default
```

Reply:
Oops, thanks! I fixed these up in #1786.